src/ts/components/subtitleoverlay.ts

```diff
@@ -34,12 +34,13 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
   private static readonly CLASS_CONTROLBAR_VISIBLE = 'controlbar-visible';
   private static readonly CLASS_CEA_608 = 'cea608';
 
+  private FONT_SIZE_FACTOR: number = 1;
   // The number of rows in a cea608 grid
-  private static readonly CEA608_NUM_ROWS = 15;
+  private CEA608_NUM_ROWS = 15 / Math.max(this.FONT_SIZE_FACTOR, 1);
   // The number of columns in a cea608 grid
-  private static readonly CEA608_NUM_COLUMNS = 32;
+  private CEA608_NUM_COLUMNS = 32 / this.FONT_SIZE_FACTOR;
   // The offset in percent for one column (which is also the width of a column)
-  private static readonly CEA608_COLUMN_OFFSET = 100 / SubtitleOverlay.CEA608_NUM_COLUMNS;
+  private CEA608_COLUMN_OFFSET = 100 / this.CEA608_NUM_COLUMNS;
```

Comment:
Nit: We might want to move these into the constructor by calling `recalculateCEAGrid` directly.